lib/zebra.h

```diff
@@ -189,9 +189,9 @@ typedef enum {
 	for (afi = AFI_IP; afi < AFI_MAX; afi++)                               \
 		for (safi = SAFI_UNICAST; safi < SAFI_MAX; safi++)
 
-#define FOREACH_AFI_SAFI_NSF(afi, safi)                                        \
-	for (afi = AFI_IP; afi < AFI_MAX; afi++)                               \
-		for (safi = SAFI_UNICAST; safi <= SAFI_MPLS_VPN; safi++)
+#define FOREACH_AFI_SAFI_NSF(afi, safi)                                                           \
+	for (afi = AFI_IP; afi < AFI_MAX; afi++)                                                  \
+		for (safi = SAFI_UNICAST; safi <= SAFI_EVPN; safi++)
```

Comment:
You may consider, to avoid missing out the bgp_gr_supported_for_afi_safi check and keeping it consolidted:
```suggestion
#define FOREACH_AFI_SAFI_NSF(afi, safi)                                                        \
	for (afi = AFI_IP; afi < AFI_MAX; afi++)                                                   \
		for (safi = SAFI_UNICAST; safi <= SAFI_EVPN; safi++)                                   \
		    if (bgp_gr_supported_for_afi_safi(afi,safi)
```